recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,9 @@
 # rust compilier specified so the rust-gnu compiler will not be used.
 rust_compiler:
   - rust
+rust_compiler_version:
+  - 1.76.0
+MACOSX_DEPLOYMENT_TARGET:   # [osx and x86_64]
+  - "10.12"                 # [osx and x86_64]
+CONDA_BUILD_SYSROOT:        # [osx and x86_64]
+  - /opt/MacOSX10.12.sdk    # [osx and x86_64]
```

Comment:
As Daniel said it's not really needed, right?

Reply:
It is to build with 1.76. I added a commit to build with 1.71.